htdocs/main.inc.php

```diff
@@ -2575,17 +2575,24 @@ function top_menu_quickadd()
 {
 	global $conf, $langs;
 
+	$tmp = getBrowserInfo($_SERVER["HTTP_USER_AGENT"]);
+	$ostype = $tmp['browseros'];
```

Comment:
ostype is already in $conf->browser->os

Reply:
@fappels good news, great, it's fixed, thank you